src/node/test/channels.cpp

```diff
@@ -501,20 +514,20 @@ TEST_CASE("Replay and out-of-order")
 
   INFO("Restart, message, resume");
```

Comment:
Please correct me if I'm wrong but I don't think we test for the auto-refreshing of the channel when the message limit has been reached. If the message limit is passed as an argument to the `Channel` constructor, this should be easy to set it to a low value in this unit test and see if the `key_exchange_init` message is produced.

Reply:
Good point, I'll add a new test for that!